packages/driver/src/cy/multi-domain/util.ts

```diff
@@ -21,7 +21,7 @@ export const correctStackForCrossDomainError = (serializedError: any, userInvoca
 export const serializeRunnable = (runnable) => {
   if (!runnable) return undefined
 
-  const fields = _.pick(runnable, ['id', '_currentRetry', 'type', 'title', 'parent', 'ctx', 'titlePath'])
```

Comment:
I don't think we still need _currentRetry, no tests are failing without it being synced.